.github/workflows/zowe-explorer-samples.yml

```diff
@@ -38,8 +38,8 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
 
-      # install yarn
-      - run: npm install -g yarn
+      # install pnpm
+      - run: npm install -g pnpm
 
-      - run: for dir in samples/*; do pushd $dir && yarn && yarn run compile && popd; done
+      - run: for dir in samples/*; do pushd $dir && pnpm i && pnpm run compile && popd; done
```

Comment:
I think this workflow should be fixed if we add the `--ignore-workspace` flag:
* `pnpm i` -> `pnpm --ignore-workspace i`